parsec/core/backend_connection/authenticated.py

```diff
@@ -367,6 +379,14 @@ async def _acquire_transport(
             self._cancel_manager_connect()
             raise
 
+        except BackendOutOfBallparkError as exc:
```

Comment:
`exc` is redefined in the except block (it's quite error prone by the way, it took me some time to figure out your "caller doesn't need to know" comment where it looked like you were re-raising the original exception ^^)